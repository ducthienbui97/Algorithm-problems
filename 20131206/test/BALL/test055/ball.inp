7 923434 927943 716622 926595
883592 868565 542431 149815 446642 260792 206971 693464 394670 90282 241493 885524 510396 185492 428129 546401 887625 392142 225765 807534 923246 849804 7505 58359 617297 633289 466369 885504 941170 844875 339497 172889 427809 849587 535032 85373 371151 333360 499403 150945 958282 489621 86503 620192 285252 464392 888087 487908 140415 
399714 945428 732572 963153 456692 276618 517799 734195 576321 119872 422282 208772 386041 934187 119929 339011 427533 61867 880635 840955 970629 551501 741783 106378 950883 450654 292853 679788 152072 956510 814282 476313 658093 59492 30851 99851 46299 66944 739915 757762 886601 279802 142555 275573 567409 897017 794344 228950 435165 
628648 179378 756496 986315 377640 982079 450764 580658 859341 151476 299772 597872 803209 883689 662095 466116 301293 422840 394139 18911 929615 904962 385827 46951 568255 994616 423369 577181 602946 692928 553241 627525 144333 122964 916254 65337 781122 718068 241373 242817 38392 884086 757670 823860 499571 765719 795758 332521 404495 
839311 823342 401230 406480 3389 329158 181303 605329 128862 909044 706945 543307 985376 890115 153240 425932 743492 856753 88468 930346 631094 522043 859287 68980 299378 193237 487362 529726 696127 355574 769592 164139 932840 716716 642840 568912 779101 986575 258425 12376 149365 441544 548072 339239 938454 732716 786549 891177 886344 93590 946525 928041 260512 157907 850165 45920 901103 711215 146928 928594 389748 242014 522272 702118 368408 190802 390013 860917 61888 439816 229094 960834 349622 428782 698759 686906 384832 19804 717739 667448 688809 730953 902567 836679 674702 510207 358751 365260 735291 215716 761285 106165 336781 357288 137040 691291 750874 753065 987182 643617 41530 876424 477787 352259 99576 267010 865436 113167 294750 703677 3824 440744 187051 600656 44165 419598 726764 319751 262055 437744 772495 815924 979402 635751 363594 552911 468589 749523 202029 15683 895086 306477 687341 675737 958522 808452 667129 83326 633513 842385 375344 238837 575199 609317 402642 580671 613958 217326 588322 259406 241190 299095 568854 557231 490487 753877 623532 507992 533291 900927 666860 168155 333368 384874 583758 410270 862334 913600 935927 889472 463212 252603 841952 28959 248957 38871 308571 804233 145604 563459 432971 166665 415885 413247 896805 7060 725193 856485 783001 874435 150606 100342 501122 268854 99588 785604 488224 687439 658482 955427 974806 807498 184228 55470 641728 567549 792808 372300 530889 131905 98270 846032 749478 756639 820258 246014 497682 655003 480085 479082 650737 371880 539259 644841 143461 619283 351425 735126 986685 581053 849490 742274 217323 900032 48977 548119 489409 896577 778836 186250 607078 373240 275546 143461 464904 375324 77797 921663 641458 512952 316120 811893 922959 922788 731356 835150 651202 688977 566383 199256 121652 581657 500751 515314 738236 652762 533636 309292 202250 273903 920245 998277 139738 376757 444359 332697 598172 690032 229296 758118 722721 276664 129378 696293 440616 311979 903574 414226 910405 436310 170792 717724 433048 870651 891939 703385 396508 837752 957193 172681 539349 976764 697092 933223 68161 435476 843700 828737 781488 131653 773194 766952 397103 516437 918338 386552 63034 714622 295382 330581 718466 986036 532766 216124 601398 346386 302674 513541 550842 513159 287591 884361 875924 900595 546219 234641 280476 356369 773257 37133 27839 696090 236483 