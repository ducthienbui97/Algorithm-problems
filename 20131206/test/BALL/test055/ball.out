18886590242094
